SIL.ReleaseTasks/SetReleaseNotesProperty.cs

```diff
@@ -130,7 +136,7 @@ private string ConvertLatestChangelog(int level, int skipUntilLevel = -1)
 							_currentIndex = _markdownLines.Length;
 						break;
 					default:
-						if (level > skipUntilLevel)
+						if ((level > skipUntilLevel) && !_urlRegex.IsMatch(currentLine))
```

Comment:
If the regex matches we don't have to process the rest of the file, so we should set:

```
_currentIndex = _markdownLines.Length;
```

Reply:
No need for this `if` if the regex matches, so I'd write this as:

```
if (_urlRegex.IsMatch(currentLine))
	_currentIndex = _markdownLines.Length;
else if (level > skipUntilLevel)
	bldr.AppendLine(currentLine);
break;
```